composer.json

```
@@ -46,7 +46,12 @@
         "sylius/grid": "self.version"
     },
     "conflict": {
-        "twig/twig": "^1.0"
+        "twig/twig": "^1.0",
+        "pagerfanta/core": "^2.0",
+        "pagerfanta/doctrine-phpcr-odm-adapter": "^2.0",
+        "pagerfanta/doctrine-dbal-adapter": "^2.0",
+        "pagerfanta/doctrine-orm-adapter": "^2.0",
+        "pagerfanta/pagefanta": "^2.0"
```

```suggestion
        "pagerfanta/core": "<3.0",
        "pagerfanta/doctrine-phpcr-odm-adapter": "<3.0",
        "pagerfanta/doctrine-dbal-adapter": "<3.0",
        "pagerfanta/doctrine-orm-adapter": "<3.0",
        "pagerfanta/pagefanta": "<3.0"
```

Conflicting only on `^2.0` would still allow 1.0 to be installed.